### ISSUE TITLE
Template not extending bootstrap but override it

### ISSUE THREAD

**Issue by pradosh987** [stisla/stisla#36]
Is there any specific reason why this template is overriding bootstrap 4 css and not extending it using scss? I mean overriding is ok but extending with scss gives more flexibility and simplicity.

**Comment by nauvalazhar** [comment]
You could be right. But, I see another view, that by overriding Bootstrap it will be easier to update, I can use CDN, I don't "touch" their code or anything else. I position Bootstrap as a framework, not as a UI kit (although they provide pre-designed components). I only need Bootstrap for layout, component base, etc. That way, I just need to override their style, no need to create it from scratch, and when there is an update it will be easy to update.

In short, I don't want to "touch" the Bootstrap code for ease of life 😆

**Comment by pradosh987** [comment]
Bootstrap has been very stable in its release cycle and breaking changes has only been in major version release so its pretty safe to extend it. I find this approach difficult to reason with simply because its to have consistency with bootstrap framework. 

Anyways your theme looks good. Keep up the good work.
